src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final Random RANDOM = new Random();
+
+    public static Ball getRandomBall() {
+        return new Ball(ColorSupplier.getRandomColor().getColorName(), RANDOM.nextInt(100));
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants